pkg/updater/service/helper/main.go

```diff
@@ -0,0 +1,123 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package main is a package that allows dd-updater
+// to execute a subset of priviledged commands
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"os/user"
+	"strconv"
+	"strings"
+	"syscall"
+	"unicode"
+)
+
+var (
+	installPath string
+	systemdPath = "/lib/systemd/system" // todo load it at build time from omnibus
+)
+
+func isValidChar(c rune) bool {
+	return unicode.IsLower(c) || c == '.' || c == '-' || c == ' '
+}
+
+func isValidString(s string) bool {
+	for _, c := range s {
+		if !isValidChar(c) {
+			return false
+		}
+	}
+	return true
+}
+
+func buildCommand(inputCommand string) (*exec.Cmd, error) {
+	if !isValidString(inputCommand) {
+		return nil, fmt.Errorf("invalid command")
+	}
+	tokens := strings.Split(inputCommand, " ")
```

Comment:
Since we're doing go <=> go let's use JSON instead of manually parsing things

Reply:
it doesn't bring any simplification with the current set of commands, we still have to enforce validation of the unit